client/my-sites/exporter/guided-transfer-card/complete-purchase-notice.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import { cartItems } from 'lib/cart-values';
+import upgradesActions from 'lib/upgrades/actions';
+import page from 'page';
+
+class CompletePurchaseNotice extends Component {
+	redirectToCart = () => {
```

Comment:
Oh whoops, I overlooked `this.props.siteSlug`. _But_, I'd recommend using `connect` to retrieve this, not passing from the parent component if possible, like so:

```js
export default connect( ( state ) => {
	return {
		siteSlug: getSiteSlug( state, getSelectedSiteId( state ) )
	};
} )( localize( CompletePurchaseNotice ) );
```

See also: 
 - [`getSelectedSiteId`](https://github.com/Automattic/wp-calypso/blob/dcf83067030e472fdc2e60355e7ebf4b8ab8528e/client/state/ui/selectors.js#L28-L36)
- [`getSiteSlug`](https://github.com/Automattic/wp-calypso/blob/dcf83067030e472fdc2e60355e7ebf4b8ab8528e/client/state/sites/selectors.js#L177-L195)

Generally I recommend `connect` as close to the component using the data as possible.